manifests/moco-nodes.pp

```diff
@@ -420,6 +420,15 @@
     $only_user_ssh       = true
     include toplevel::server::beetmoverscriptworker
 }
+#
+# https://github.com/mozilla-mobile workers.
+node /^mobile-beetmover-2\.srv\.releng\.use1\.mozilla\.com$/ {
```

Comment:
How about `mobil-beetmover-dev\d+`? (Both pushapk and signingscript removed the 'e' at the end of mobile to maintain 2 digits

Reply:
Much better!